queue_job/README.rst

```diff
@@ -105,7 +105,7 @@ Configuration
   [options]
   (...)
   workers = 6
-  server_wide_modules = web,queue_job
+  server_wide_modules = web,queue_job,base_sparse_field
```

Comment:
IMO this is not needed when the fields is imported from the module like in v14 https://github.com/OCA/queue/blob/14.0/queue_job/models/queue_job.py#L12 and is not needed at all on v15 AFAIK since a custom field is used instead https://github.com/OCA/queue/blob/15.0/queue_job/fields.py#L14

Am I missing anything?

Reply:
On v14 was needed for sure. If not, we got this error:

![imagen](https://user-images.githubusercontent.com/7165771/184898605-0a0fd4cd-0760-4dc0-bf9c-6e817bde0c87.png)

For v15 indeed the sparse field is gone, so we can close this.

Thanks for being aware of it.

Reply:
Welcome :) 
I'll comment on the v14 PR for the other detail.